tests/provider_install_qrcode_test.go

```diff
@@ -72,7 +72,8 @@ var _ = Describe("kairos qr code install", Label("provider", "provider-qrcode-in
 		Expect(err).ToNot(HaveOccurred(), device)
 
 		By("registering with a screenshot")
-		err = register("fatal", fileName, "./assets/config.yaml", strings.TrimSpace(device), true)
+		// pass a config that auto reboots after install as we cannot know when the machine has finished
+		err = register("fatal", fileName, "./assets/config_with_reboot.yaml", strings.TrimSpace(device), true)
```

Comment:
The file name above has a typo: `_rebot` vs `reboot`. This file here doesn't exist. How did this work?

Reply:
It worked on my branch where I had moment of brilliance and was able to spell reboot .

No wonder it's still failing. You just saved me a couple of hours of troubleshooting lol